client/pages/offer-thread.jsx

```diff
@@ -62,30 +86,33 @@ class OfferThread extends React.Component {
     })
       .then(response => response.json())
       .then(reply => {
-
       })
       .catch(err => console.error(err));
     this.setState({ reply: '' });
   }
 
+  componentDidUpdate() {
+    this.scrollToBottom();
+  }
```

Comment:
See my other comment. We don't want to just always "scroll to bottom" we can do it specifically when the chat list gets updated.